app/controllers/admin/worldwide_offices_controller.rb

```diff
@@ -17,8 +17,17 @@ def edit
   end
 
   def update
-    worldwide_office_params[:service_ids] ||= []
-    if @worldwide_office.update(worldwide_office_params)
+    update_params = worldwide_office_params
+
+    update_params[:service_ids] ||= []
+
+    if update_params.dig(:contact_attributes, :contact_numbers_attributes)
+      update_params.dig(:contact_attributes, :contact_numbers_attributes).each do |_id, contact_number_attribute|
```

Comment:
We could store this in a variable, rather than `dig`ging twice, i.e.

```
contact_numbers_attributes = update_params.dig(:contact_attributes, :contact_numbers_attributes)
if (contact_number_attributes)...
```

Reply:
Applied :)

Reply:
@callumknights I don't think this change got pushed to GitHub?  Code looks the same as before.  🤔

Reply:
If you're fixing the ``undefined method `empty?' for nil `` issue, this could also be simplified to `update_params.dig(:contact_attributes, :contact_numbers_attributes)&.each`